### ISSUE TITLE
Cocoadocs xcodebuild error

### ISSUE THREAD

**Issue by ghodasara** [CocoaPods/cocoadocs.org#415]
Hey,

I've documented my pod, https://cocoapods.org/pods/ForecastIO, and the latest version is available on Cocoapods, but the documentation isn't showing up on Cocoadocs. I pinged the API to check for errors, and I got this back:

```
report_error({  
   "message":"/Users/cocoadocs/.rvm/gems/ruby-2.1.3/gems/jazzy-0.4.1/lib/jazzy/SourceKitten/bin/sourcekitten [\"doc\"]\n\nRunning xcodebuild\n\nCould not parse compiler arguments from `xcodebuild` output.\n\nPlease confirm that `xcodebuild` is building a Swift module.\n\nSaved `xcodebuild` log file: /var/folders/sm/8nr9m40j0yx6bpc_qhsnk36h0000gp/T/xcodebuild-6E25DE8B-A561-4DC0-8093-0F0F718F0E35.log\n\nFailed to generate documentation\n",
   "trace":[  
      "/Users/cocoadocs/.rvm/gems/ruby-2.1.3/gems/jazzy-0.4.1/lib/jazzy/executable.rb:36:in `execute_command'",
      "/Users/cocoadocs/.rvm/gems/ruby-2.1.3/gems/jazzy-0.4.1/lib/jazzy/sourcekitten.rb:140:in `run_sourcekitten'",
      "/Users/cocoadocs/.rvm/gems/ruby-2.1.3/gems/jazzy-0.4.1/lib/jazzy/doc_builder.rb:57:in `block in build'",
      "/Users/cocoadocs/.rvm/gems/ruby-2.1.3/gems/jazzy-0.4.1/lib/jazzy/doc_builder.rb:55:in `chdir'",
      "/Users/cocoadocs/.rvm/gems/ruby-2.1.3/gems/jazzy-0.4.1/lib/jazzy/doc_builder.rb:55:in `build'",
      "/Users/cocoadocs/cocoadocs.org/./cocoadocs.rb:439:in `document_spec'",
      "/Users/cocoadocs/cocoadocs.org/./cocoadocs.rb:529:in `document_spec_at_path'",
      "/Users/cocoadocs/cocoadocs.org/./cocoadocs.rb:165:in `block in doc'",
      "/Users/cocoadocs/cocoadocs.org/./cocoadocs.rb:160:in `each'",
      "/Users/cocoadocs/cocoadocs.org/./cocoadocs.rb:160:in `doc'",
      "/Users/cocoadocs/cocoadocs.org/./cocoadocs.rb:232:in `block in cocoadocs_url'",
      "/Users/cocoadocs/.rvm/rubies/ruby-2.1.3/lib/ruby/2.1.0/open-uri.rb:153:in `open_uri'",
      "/Users/cocoadocs/.rvm/rubies/ruby-2.1.3/lib/ruby/2.1.0/open-uri.rb:704:in `open'",
      "/Users/cocoadocs/.rvm/rubies/ruby-2.1.3/lib/ruby/2.1.0/open-uri.rb:34:in `open'",
      "/Users/cocoadocs/cocoadocs.org/./cocoadocs.rb:229:in `cocoadocs_url'",
      "/Users/cocoadocs/cocoadocs.org/./cocoadocs.rb:180:in `cocoadocs'",
      "/Users/cocoadocs/cocoadocs.org/./cocoadocs.rb:115:in `initialize'",
      "/Users/cocoadocs/cocoadocs.org/./cocoadocs.rb:558:in `new'",
      "/Users/cocoadocs/cocoadocs.org/./cocoadocs.rb:558:in `<main>'"
   ]
})
```

I can run jazzy locally with my .jazzy.yaml config file, and I don't experience any errors and I get my docs exactly as I want them. Unfortunately, this error is hard to debug without being able to view the .log file that was generated. Any suggestions on how to proceed from here?

**Comment by orta** [comment]
Might be worth trying the preview command documented in the README for CocoaDocs, so far I've not had time to add support for jazzy config files, you could also look into that too

**Comment by ghodasara** [comment]
I think I've identified the issue. I did `pod lib create` to create my pod, which generates a _Pods.xcodeproj symlink at the root to Example/Pods/Pods.xcodeproj. Running jazzy at the root on _Pods.xcodeproj breaks some of the paths involved in building the project, so it fails. Running jazzy on Example/Pods/Pods.xcodeproj is successful. Running the preview command with cocoadocs.rb has the same issue with broken paths as running jazzy at the root on _Pods.xcodeproj. I think I can solve this by restructuring my pod, so I'll close this issue, but I'm surprised that the default pod setup is incompatible with Cocoadocs.

**Comment by orta** [comment]
Hrm, I don't think this is it, ( plus CocoaDocs doesn't use that xcodeproj bundle ) - I was thinking of looking at some CD tasks today, so will take a pick a at this too.

**Comment by ghodasara** [comment]
I went ahead and restructured my pod and it worked. I deleted _Pods.xcodeproj and moved Example/Pods/Pods.xcodeproj to the root and fixed the links within the project. After this the documentation showed up properly on CocoaDocs.

**Comment by orta** [comment]
Thanks for the follow up @ghodasara - I also fixed a few jazzy issues today too, so I'm glad it's working!

**Comment by orta** [comment]
Should be fixed with #422 I think
